internal/resources/fabric/precision_time/resource_test.go

```diff
@@ -0,0 +1,268 @@
+package precision_time
```

Comment:
We encourage black box testing rather than tying tests to private implementation details.  The `_test` package is a Go idiom that allows you to keep tests in the same directory as the code under test, but force those tests to use the code as a consumer would.  (Ultimately your team are the owners of this code and the decision as to whether to use black box testing for each resource is entirely up to you.)

```suggestion
package precision_time_test
```